packages/odyssey/src/scss/abstracts/_fonts.scss

```diff
@@ -13,28 +13,43 @@
 // limitations under the License.
 
 @font-face {
-  font-family: 'Proxima Nova';
+  font-family: 'whyte-trial';
   font-style: normal;
-  font-weight: 300;
-  src:
-    url('/font/ProximaNova-Light.woff2') format('woff2'),
-    url('/font/ProximaNova-Light.woff') format('woff');
+  font-weight: normal;
+  src: url('/font/WhyteTrial-Book.woff2') format('woff2'), url('/font/WhyteTrial-Book.woff') format('woff');
```

Comment:
Saw the trial files above - we can't ship any licensed binaries with ODS, so we'll have to remove those.

Let's keep these declarations in place for now, just not the binaries - at least until there's another package to do the declarations appropriately.

Reply:
Ill be sure to remove this before merge. If I remove the binary right now the deployment will render with system fonts (which we don't want for review purposes)

Reply:
Once we have the official font, let's make sure anyone reviewing is going to have it on their system.